pkg/build/generator/generator.go

```diff
@@ -93,6 +100,31 @@ func (g *BuildGenerator) Instantiate(ctx kapi.Context, request *buildapi.BuildRe
 	if err != nil {
 		return nil, err
 	}
+
+	// Fetch the secrets associated with the BuilderServiceAccount.
+	// These secrets will be used for push/pull authentication
+	// FIXME: Add support for SourceSecret
+	if sa, err := g.ServiceAccounts.ServiceAccounts(bc.Namespace).Get(bootstrappolicy.BuilderServiceAccountName); err != nil {
```

Comment:
make the default service account name a field in BuildGenerator, initialized to BuilderServiceAccountName for now